tiny_dnn/layers/max_unpooling_layer.h

```diff
@@ -151,7 +148,7 @@ class max_unpooling_layer : public feedforward_layer<Activation> {
       in2outmax_;  // mapping max_index(out) => in (1:1)
   };
 
-  std::vector<max_unpooling_layer_worker_specific_storage>
+  max_unpooling_layer_worker_specific_storage
     max_unpooling_layer_worker_storage_;
```

Comment:
I think struct name `max_unpooling_layer_worker_specific_storage` is too long. Since it's an inner class, it can be renamed to `worker_specific_storage`. The member variable `max_unpooling_layer_worker_storage_` can also be renamed to just `worker_storage_`.